bundles/net.tourbook.cloud/src/net/tourbook/cloud/strava/PrefPageStrava.java

```diff
@@ -173,15 +174,15 @@ private void createUI_20_AccountInformation(final Composite parent) {
       {
```

Comment:
To have a common UI between all cloud providers, I'm missing a web page url when a user is not yet authorized...

![mt-cloud-strava](https://user-images.githubusercontent.com/1283445/108589228-c0e02580-735d-11eb-8ec9-69b384032e3e.png)

... like in the other cloud pref pages

![mt-cloud-other](https://user-images.githubusercontent.com/1283445/108589229-c178bc00-735d-11eb-8a03-fdf8e056d548.png)

Reply:
I will add that in another PR

Reply:
Fixed in [PR](https://github.com/wolfgang-ch/mytourbook/pull/305)